Dockerfile

```diff
@@ -32,18 +32,23 @@ RUN adduser --disabled-login --disabled-password ubuntu --ingroup ubuntu
 # Give Ownership of sandbox env to sandbox group and user
 RUN chown -R $CODEJAIL_TEST_USER:$CODEJAIL_GROUP $CODEJAIL_TEST_VENV
 
-# Clone Codejail Repo
-ADD . ./codejail
 WORKDIR /codejail
 
+# Clone Requirement files
+COPY ./requirements/sandbox.txt ./codejail/requirements/sandbox.txt
```

Comment:
Improvement to the image building process. Requirements Cache wouldn't be busted this way if there are code changes but no change to requirements file.